bats/tests/helpers/vm.bash

```diff
@@ -74,7 +74,6 @@ container_engine_info() {
 
 docker_context_exists() {
     run docker_exe context ls -q
-    assert_success
```

Comment:
Why did you delete this line? The command needs to succeed before we check that it produces the correct output.

Under which conditions can the command exit with a non-0 return code, but still show the right context?